### DISCUSSION TITLE
None of my repos are loading

### DISCUSSION THREAD

**Discussion by DenisRegan** [community/community#62916] [Repositories]
### Select Topic Area

Bug

### Body

Hi . None of repos are working and all I can see is loading icons spinning . Any repos are greyed out/the files are not visible

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @DenisRegan, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
